lib/mail_room/configuration.rb

```diff
@@ -2,16 +2,21 @@ module MailRoom
   # Wraps configuration for a set of individual mailboxes with global config
   # @author Tony Pitale
   class Configuration
-    attr_accessor :mailboxes, :daemonize, :log_path, :pid_path
+    attr_accessor :mailboxes, :daemonize, :log_path, :pid_path, :quiet
 
     # Initialize a new configuration of mailboxes
     def initialize(options={})
       self.mailboxes = []
+      self.quiet = options.fetch(:quiet, false)
 
       if options.has_key?(:config_path)
-        config_file = YAML.load_file(options[:config_path])
+        begin
+          config_file = YAML.load_file(options[:config_path])
 
-        set_mailboxes(config_file[:mailboxes])
+          set_mailboxes(config_file[:mailboxes])
+        rescue => e
```

Comment:
This only rescues from the configuration portion, nothing to do with _running_ MailRoom.